frontend/src/components/PrivateRoute.js

```diff
@@ -1,16 +0,0 @@
-import React from "react";
-import {connect} from "react-redux";
-import {Redirect, Route} from "react-router-dom";
-
-const PrivateRoute = ({isAuthenticated, children, ...rest}) => (
```

Comment:
I'm not sure you had to change all these PrivateRoutes into PrivateNavigate.

Reply:
Maybe not, but I couldn't fix examples of how one could use something like `PrivateRoutes` in the new `Route` component. It also seems to be the more correct way. For example: <https://dev.to/iamandrewluca/private-route-in-react-router-v6-lg5>

Reply:
Not as much the renaming than the practicality of having a wrapper to encompass the route instead of having just a wrapper to the auth. Looks like you did your research and Its ok to change it, but it seem to be a step backward in code readability and simplicity. I don't see how this element prop fit in the react way of doing things.

Reply:
I believe it's because React developers prefers if the children of `Route` are `Route`s. Otherwise, they will have to parse the `children` prop to figure out if it's a `<Route>` or `<div>, etc.